src/components/ValueChange/ValueChange.js

```diff
@@ -10,13 +10,14 @@ const Change = {
 
 const notChanged = ['texas-rose', 'lock-small']
 
-const ValueChange = ({ className, change, render }) => {
+const ValueChange = ({ className, change, render, label }) => {
```

Comment:
**[1]** `ValueChange` is render-agnostic and should be only aware of **changed value**. If some kind of element should be rendered close it, then just make it as a sibling